thrift/linkattacher/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2016 Istituto Italiano di Tecnologia iCub Facility & RBCS Department
+# Authors: Yeshasvi Tirupachuri
+# CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+
+cmake_minimum_required(VERSION 2.8.11)
+
+set(LIBRARY_TARGET_NAME gazebo_yarp_rpc_linkattacher)
+
+yarp_idl_to_dir(linkattacher_rpc.thrift ${CMAKE_CURRENT_SOURCE_DIR}/autogenerated LARPC_SRC LARPC_HEADERS LARPC_INCLUDES)
```

Comment:
@traversaro What is the policy on idl? Committing the generated sources as in this case?

Reply:
I would say yes, because for the rest of the thrift in the repo we do that. If we swicht to generate it on the fly, let's do it for all the thrift file in the repo.

Reply:
So, no changes here